include/mixer/mixer.hh

```diff
@@ -11,33 +11,59 @@
 #include <memory>
 #include <mutex>
 
+/** mixer namespace
+ *  Contains the mixer class.
+ */
 namespace mixer {
-
-//  Mixer class takes in a list of input pipes, extracts the audio packet from
-//  each of them. It then produces a new audio packet which is the sum of the
-//  previous audio packets.
+/** Mixer class
+ *  Takes in a list of input pipes, extracts the audio packet from
+ *  each of them. It then produces a new audio packet which is the sum of the
+ *  previous audio packets.
+ */
 template <std::size_t num_banks> class Mixer {
   private:
+    // Array of pipes for the input to the class
     std::array<Pipe<Audio>, num_banks> &input_pipes;
+    // Single pipe as output for the class
     Pipe<Audio> &output_pipe;
 
+    // Not sure what this does, can't find any other reference to it
     bool run_thread{};
```

Comment:
I don't know what this does, I don't see any other reference to it in the file.

Reply:
Probably left over from before we switched to using wait_for()

Reply:
Good idea to remove it then imo. Why doesn't SA or the compiler complain about an unused variable though.